lib/ZumoOta/src/BootloaderCom.h

```diff
@@ -193,13 +200,14 @@ class BootloaderCom
     /** 
      *Specifies the current memory/page address for flashing the firmware of the Zumo program flash.
      */
-    uint16_t m_currWriteMemAddr;
+    uint16_t static m_currWriteMemAddr;

```

Comment:
unused remove

Reply:
It will be taken care of by the next commit.